packages/account/src/Components/poo-rejected/rejected.jsx

```diff
@@ -0,0 +1,23 @@
+import { Button, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import React from 'react';
+import PooRejectedIcon from '../../Assets/ic-poo-rejected.svg';
+
+export const POORejetced = ({ onTryAgain }) => {
+    return (
+        <div className='proof-of-ownership__container'>
+            <PooRejectedIcon className='icon' size={128} />
```

Comment:
```suggestion
import { Button, Text, Icon } from '@deriv/components';
import { localize } from '@deriv/translations';
import React from 'react';

export const POORejetced = ({ onTryAgain }) => {
    return (
        <div className='proof-of-ownership__container'>
            <Icon className='icon' icon='IcPooRejected' size='128' />
```
It seems like ic-poo-rejected.svg added twice here and in the main pool of icons in packages/components/src/components/icon/common/ic-poo-rejected.svg, right? 🙂

Reply:
@maryia-binary, there was a reason I did this five months ago, but I just can't seem to remember it. I'll change it to your suggestion until that reason shows up. Hopefully it doesn't